src/content/content.component.ts

```diff
@@ -17,97 +18,58 @@ import { BUTTON_TYPE } from '@shared-components/button/button.component';
 export class ContentComponent implements OnInit {
   value = '';
   SORT = SORT;
-  sortType: SORT = SORT.ASC;
   DISPLAY_TYPE = DISPLAY_TYPE;
   BUTTON_TYPE = BUTTON_TYPE;
   type: DISPLAY_TYPE = DISPLAY_TYPE.LIST;
   onlyFavoriteMovie = false;
+  allMovies: Movie[] = [];
   page: PageEvent = {
     pageIndex: DEFAULT_PAGE_INDEX,
     pageSize: DEFAULT_PAGE_SIZE,
     length: this.allMovies.length,
   };
   managedMovies: Movie[] = [];
 
-  constructor(public apiService: ApiService,
-              public moviesService: MoviesService,
-              public snackBar: MatSnackBar) {
-              }
-
+  constructor(public snackBar: MatSnackBar,
+              public movieFacade: MovieFacade,
+              public uiFacade: UiFacade) {
+  }
 
   ngOnInit(): void {
-    this.moviesService.setMoviesFromLocalStorage();
-    this.managedMovies = this.allMovies;
+    this.movieFacade.allMovies$
+      .pipe(untilDestroyed(this))
+      .subscribe(allMovies => this.allMovies = allMovies);
+    this.movieFacade.managedMovies$
```

Comment:
to daj do osobnej metody z deskryptywna nazwa, bo pozniej w ngOnInit potrafi byc 10 takich subscribe i idzie sie pogubic d;